Clappr.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = "Clappr"
-  s.version          = "0.7.5"
-  s.summary          = "An extensible media player for iOS"
+  s.version          = "0.9.0"
```

Comment:
What is the reason to change the framework version?

Reply:
Hey guys,
I asked the same thing on a [previous PR](https://github.com/clappr/clappr-ios/pull/163/files/320efe7f0c7a3da3758f4cbb6791d94de5d7f19f), but it got accepted and merged without an answer. 😅

Reply:
I agree with you guys. In tvOS we didn't publish Clappr in cocoapods, we just used a fixed version in podspec. I think we can change the version to `0.7.5` to fix this change and upgrade it to `0.8.0` when we launch the next version.

What do you think?

Reply:
Yep! I would like to keep the current version (`0.7.5`), then we can discuss the version of the next release.

Reply:
Done! :D